soroban-auth/src/public_types.rs

```diff
@@ -75,6 +75,15 @@ impl From<&AccountId> for Identifier {
     }
 }
 
+impl From<Address> for Identifier {
```

Comment:
Could you also add `From<&Address> for Identifier` too? These elements are cheap to clone.

Reply:
Added.

Reply:
Thanks!!!